README.md

```diff
@@ -27,6 +27,9 @@ If you need a version from another branch, add the `ref` argument in the above f
 
 ## You are welcome to contribute!
 
+Please have a look at our [contributing guide](https://inbo.github.io/watina/CONTRIBUTING.html)!
```

Comment:
@florisvdh it is slightly better to reference `.github/CONTRIBUTING.md`, as that will link to the GitHub file on GitHub and the html file in the pkgdown website.

Reply:
Aha, that's great - I was a bit in doubt so I chose the html file. I'll update that.